### DISCUSSION TITLE
Multi-state survival regression models

### DISCUSSION THREAD

**Discussion by joshlk** [CamDavidsonPilon/lifelines#1214] [General]
Hi,

I'm quite new to survival regression, firstly I would like to say that, I love the package and its very well documented. A great intro to the subject 😃 .

I'm currently working on a problem which has multiple-states or stages. The stages are linear so that A always predates B and B always predates C ect (A->B->C->...). Is lifetimes suited to this kind of problem and do you have examples? Are you looking to support this type of regression?

Thanks,
Josh

**Comment by CamDavidsonPilon** [comment]
Hi @joshlk, I think multi-state models are really cool, but not something lifelines will support anytime soon! (Too much other work atm to think about it)
